CONTRIBUTING.md

```diff
@@ -9,3 +9,24 @@ https://qiskit.org/documentation/contributing_to_qiskit.html
 
 In addition to the general guidelines there are specific details for
 contributing to qiskit_sphinx_theme, these are documented below.
+
+# Folder Structure
```

Comment:
The heading levels seem a bit off to me - I'd have thought this was a level 2, under the top-level "Contributing" heading at the top of the page?